src/osd/OSD.cc

```diff
@@ -8556,11 +8556,11 @@ void OSD::handle_op(OpRequestRef& op, OSDMapRef& osdmap)
     return;
   }
 
-  PG *pg = get_pg_or_queue_for_pg(pgid, op);
-  if (pg) {
+  PGRef pg = get_pg_or_queue_for_pg(pgid, op);
+  if (pg.get()) {
     op->send_map_update = share_map.should_send;
     op->sent_epoch = m->get_map_epoch();
-    enqueue_op(pg, op);
+    enqueue_op(pg.get(), op);
```

Comment:
How annoying would it be to update enqueue_op to take a PGRef?  It's going to build a PGRef anyway, might as well do it here and avoid the .get() nonsense.